### PR TITLE
Ansible

### PR DESCRIPTION
**Pull request by gregcorbett**
adds an accounting.yaml file which can be used to deploy the service on kubernetes infrastructure

### CODE REVIEW

File: accounting.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+---
+mysql:
+apiVersion: v1
+kind: Service
+metadata:
+  name: accounting-mysql
+  labels:
+    app: accounting
+spec:
+  clusterIP: 10.254.10.21
```
Comment (tofu-rocketry) [inline accounting.yaml]: This IP appears a lot - is it possible to have it as a single config option?

It should also probably not be a specific IP address in the repo.

File: apel_rest/urls.py [hunk 2]
```diff
@@ -12,5 +12,7 @@
 
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/v1/cloud/record$', CloudRecordView.as_view()),
-    url(r'^api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view())
+    url(r'^api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view()),
+    url(r'^accounting-server/api/v1/cloud/record$', CloudRecordView.as_view()),
```
Comment (tofu-rocketry) [inline apel_rest/urls.py]: What is the reason for duplicating the API under different parent branches?

Could this be a config option?

File: accounting.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+             value: apel
+           - name: MYSQL_PASSWORD
+             value: PutYourPassHere
+          volumeMounts:
+           - name: accounting-mysql
+             mountPath: /var/lib/mysql
+      volumes:
+        - name: accounting-mysql
+          cinder:
+            volumeID: 44de1b82-2642-4753-915a-54d3d6b54fea
```
Comment (tofu-rocketry) [inline accounting.yaml]: The value of this `volumeID` seems like a infrastructure specific thing that shouldn't be in the repo.

**Comment by tofu-rocketry** [comment]
`ssl.conf` seems to be removed by this. Is that correct?

File: accounting.yaml (hunk 1 above)
Comment (gregcorbett) [inline accounting.yaml]: I'm not sure about as a single config option, although agreed that would be preferable

File: apel_rest/urls.py (hunk 2 above)
Comment (gregcorbett) [inline apel_rest/urls.py]: there isn't a good reason, removed.

File: accounting.yaml (hunk 3 above)
Comment (gregcorbett) [inline accounting.yaml]: Yes, removed in the same commit as the IP addresses

**Comment by gregcorbett** [comment]
It's intentional.

In terms of enabling HTTPS, I'm told kubernetes would handle this, So this server doesn't need a specific SSL.conf. 

In terms of certificate verification, some parts of SSL.conf will probably have to come back.

**Pull request merged**